cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperator.java

```diff
@@ -194,6 +189,18 @@ protected KafkaMirrorMaker2Status createStatus(KafkaMirrorMaker2 ignored) {
         return new KafkaMirrorMaker2Status();
     }
 
+    /**
+     * Deletes the ClusterRoleBinding which as a cluster-scoped resource cannot be deleted by the ownerReference
+     *
+     * @param reconciliation    The Reconciliation identification
+     * @return                  Future indicating the result of the deletion
+     */
+    @Override
+    protected Future<Boolean> delete(Reconciliation reconciliation) {
```

Comment:
should be the method name more clear about what it does? It's written in the Javadoc but then it's named just `delete` which is not so explicit.

Reply:
I thought about it as well. But it implements/overrides the method from upper class. Technically, the method is responsible for deleting everything that is not deleted through garbage collection. But as it goes, it right now deletes only the cluster role binding. So I think the name is ok.

Reply:
I missed the override, so there is not that much we can do.